Parsers/ASimAuthentication/Parsers/vimAuthenticationVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,125 @@
+Parser:
+  Title: ASIM Authentication parser for VMware Carbon Black Cloud
+  Version: '0.1.0'
+  LastUpdated: Oct 09 2023
+Product:
+  Name: VMware Carbon Black Cloud
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: VMware Carbon Black Cloud documentation
+  Link: 
+    https://developer.carbonblack.com/reference/carbon-black-cloud/data-forwarder/schema/latest/endpoint.event-1.0.0/
+    https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Authentication normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
+ParserName: vimAuthenticationVMwareCarbonBlackCloud
+EquivalentBuiltInParser: _Im_Authentication_VMwareCarbonBlackCloud
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresultdetails_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: targetusername_has_any
+    Type: string
+    Default: '*'
+  - Name: targetappname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: actorusername_has_any
+    Type: string
+    Default: '*'
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: srchostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvchostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (starttime: datetime=datetime(null), endtime: datetime=datetime(null), eventtype_in: dynamic=dynamic([]), eventresultdetails_in: dynamic=dynamic([]), eventresult: string='*', targetusername_has_any: string='*', targetappname_has_any: dynamic=dynamic([]), actorusername_has_any: string='*', srcipaddr_has_any_prefix: dynamic=dynamic([]), srchostname_has_any: dynamic=dynamic([]), targetipaddr_has_any_prefix: dynamic=dynamic([]), dvcipaddr_has_any_prefix: dynamic=dynamic([]), dvchostname_has_any: dynamic=dynamic([]), disabled: bool = false) {
```

Comment:
targetusername_has_any
actorusername_has_any
 both should be dynamic

Reply:
Updated

Reply:
Done